README.md

```diff
@@ -14,7 +14,18 @@
 This driver is an extension of the [Selenium](http://docs.seleniumhq.org/) C# client. It has 
 all the functionalities of the regular driver, but add Appium-specific methods on top of this.
 
-## v5 Release Candidate 
+## Compatibility Matrix
+
+The Appium .NET Client depends on [Selenium .NET binding](https://www.nuget.org/packages/Selenium.WebDriver), thus the Selenium .NET binding update might affect the Appium .NET Client behavior. 
+For example, some changes in the Selenium binding could break the Appium client.
+
+|Appium .NET Client| Selenium Binding	| .NET Version |
```

Comment:
Is this the minimum supported .NET version?

Reply:
Not really, you will need to have one of those specific versions installed.
You can still have .NET 8 installed on the machine in addition to .NET 6. But in the end it will use .NET 6